index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head> 
+        <title> Salmon Cookies</title>
+         <meta charset="utf-8">
+         <link rel="stylesheet" href="css/reset.css">
+         <link rel="stylesheet" href="css/style.css">
+         <link href="https://fonts.googleapis.com/css?family=Macondo|Slabo+27px" rel="stylesheet">
+         <link href="https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz" rel="stylesheet">
+    </head>
+         <body>
+            <nav>
+                <section class="LargeSalmon">
+                    <img src= "C:\Users\Ray\codefellows\201\cookielab\cookie-stand\salmon.png" alt="Photograph of large odd salmon" title= "large salmon">
```

Comment:
Make sure this link is a relative link. Currently, this will not work anywhere except your computer.